README.md

```diff
@@ -193,19 +175,46 @@ const styles = StyleSheet.create({
 |:---:|:---:|
 | iPhone Horizontal Portrait | iPhone Horizontal Landscape  |
 
-## Example Nested Grid (itemContainerStyle)
-
-If you try to use nested grid with different number of items, you could use itemContainerStyle to change the style.
-Please take a look on the follow example.
-
-```
+## SectionGrid Example
+```javascript
 import React, { Component } from 'react';
 import { StyleSheet, View, Text } from 'react-native';
-import GridView from 'react-native-super-grid';
+import { FlatGrid } from 'react-native-super-grid';
+
```

Comment:
This is meant to be an example for the SectionGrid, but it's FlatGrid code that's displayed here.

Reply:
Yea. My bad. Fixing it shortly.